src/server/socket.js

```diff
@@ -124,6 +124,10 @@ SocketServer.prototype.reloadSimHost = function () {
     this._emitTo(SIM_HOST, 'refresh');
 };
 
+SocketServer.prototype.rethemeSimHost = function () {
```

Comment:
What is our plan to get this thing invoked? 

Can the updateTheme() method in simulator.js call it?